samples/matter/bridge/src/bridged_device_data_provider.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include <app/util/attribute-storage.h>
+
+class BridgedDeviceDataProvider {
+public:
+	using UpdateAttributeCallback = void (*)(BridgedDeviceDataProvider *dataProvider, chip::ClusterId clusterId,
```

Comment:
```suggestion
	using UpdateAttributeCallback = void (*)(BridgedDeviceDataProvider &dataProvider, chip::ClusterId clusterId,
```
suggest using references whenever nulls don't make sense, to avoid redundant null-checks.